clustered/build.gradle

```diff
@@ -15,14 +15,13 @@
  */
 
 ext {
-    terracottaPlatformVersion = '5.0.0.beta'
-    terracottaApisVersion = '1.0.0.beta'
+    terracottaPlatformVersion = '5.0.3.beta'
+    terracottaApisVersion = '1.0.3.beta'
     terracottaCoreVersion = '5.0.0-beta2'
-    coordinatorVersion = terracottaPlatformVersion
     clusteredMapVersion = terracottaPlatformVersion
     offheapResourceVersion = terracottaPlatformVersion
     entityApiVersion = terracottaApisVersion
-    entityTestLibVersion = terracottaApisVersion
+    entityTestLibVersion = '1.0.3.beta'
```

Comment:
Are all these changes intentional... and can anyone explain to me what the system is here?